[I 2024-05-15 08:11:42.223 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2024-05-15 08:11:42.225 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2024-05-15 08:11:42.226 ServerApp] jupyterlab | extension was successfully linked.
[I 2024-05-15 08:11:42.227 ServerApp] jupyterlab_jupytext | extension was successfully linked.
[I 2024-05-15 08:11:42.228 ServerApp] notebook | extension was successfully linked.
[I 2024-05-15 08:11:42.330 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-05-15 08:11:42.337 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-05-15 08:11:42.338 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2024-05-15 08:11:42.338 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2024-05-15 08:11:42.339 LabApp] JupyterLab extension loaded from /opt/jupyter-venv/lib/python3.12/site-packages/jupyterlab
[I 2024-05-15 08:11:42.339 LabApp] JupyterLab application directory is /opt/jupyter-venv/share/jupyter/lab
[I 2024-05-15 08:11:42.339 LabApp] Extension Manager is 'pypi'.
[I 2024-05-15 08:11:42.353 ServerApp] jupyterlab | extension was successfully loaded.
[W 2024-05-15 08:11:42.353 ServerApp] [Jupytext Server Extension] Async contents managers like AsyncLargeFileManager are not supported at the moment (https://github.com/mwouts/jupytext/issues/1020). We will derive a contents manager from LargeFileManager instead.
[I 2024-05-15 08:11:42.353 ServerApp] [Jupytext Server Extension] Deriving a JupytextContentsManager from LargeFileManager
[I 2024-05-15 08:11:42.354 ServerApp] jupyterlab_jupytext | extension was successfully loaded.
[I 2024-05-15 08:11:42.355 ServerApp] notebook | extension was successfully loaded.
[I 2024-05-15 08:11:42.355 ServerApp] Serving notebooks from local directory: /media/creativeucon/D/repos/uconcreative/accent-coach/llmware
[I 2024-05-15 08:11:42.355 ServerApp] Jupyter Server 2.14.0 is running at:
[I 2024-05-15 08:11:42.355 ServerApp] http://localhost:8888/tree?token=ad58e433b19c143dc0c13530e3e9477045604ef0414e02fc
[I 2024-05-15 08:11:42.355 ServerApp]     http://127.0.0.1:8888/tree?token=ad58e433b19c143dc0c13530e3e9477045604ef0414e02fc
[I 2024-05-15 08:11:42.355 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2024-05-15 08:11:42.369 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///home/creativeucon/.local/share/jupyter/runtime/jpserver-5083-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/tree?token=ad58e433b19c143dc0c13530e3e9477045604ef0414e02fc
        http://127.0.0.1:8888/tree?token=ad58e433b19c143dc0c13530e3e9477045604ef0414e02fc
[I 2024-05-15 08:11:42.379 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[W 2024-05-15 08:20:00.730 ServerApp] Notebook examples/Datasets/build_aws_transcript_dataset.ipynb is not trusted
[W 2024-05-15 08:20:01.267 ServerApp] Notebook examples/Datasets/build_aws_transcript_dataset.ipynb is not trusted
[I 2024-05-15 08:20:06.368 ServerApp] Kernel started: 60e5df90-3bc7-480e-9648-0b8d933fd57f
[I 2024-05-15 08:20:06.717 ServerApp] Connecting to kernel 60e5df90-3bc7-480e-9648-0b8d933fd57f.
[I 2024-05-15 08:20:06.730 ServerApp] Connecting to kernel 60e5df90-3bc7-480e-9648-0b8d933fd57f.
[I 2024-05-15 08:20:06.743 ServerApp] Connecting to kernel 60e5df90-3bc7-480e-9648-0b8d933fd57f.
[I 2024-05-15 08:20:18.291 ServerApp] Interrupted...
[I 2024-05-15 08:20:47.445 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2024-05-15 08:20:47.447 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2024-05-15 08:20:47.448 ServerApp] jupyterlab | extension was successfully linked.
[I 2024-05-15 08:20:47.448 ServerApp] jupyterlab_jupytext | extension was successfully linked.
[I 2024-05-15 08:20:47.450 ServerApp] notebook | extension was successfully linked.
[I 2024-05-15 08:20:47.552 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-05-15 08:20:47.559 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-05-15 08:20:47.560 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2024-05-15 08:20:47.560 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2024-05-15 08:20:47.561 LabApp] JupyterLab extension loaded from /opt/jupyter-venv/lib/python3.12/site-packages/jupyterlab
[I 2024-05-15 08:20:47.561 LabApp] JupyterLab application directory is /opt/jupyter-venv/share/jupyter/lab
[I 2024-05-15 08:20:47.561 LabApp] Extension Manager is 'pypi'.
[I 2024-05-15 08:20:47.575 ServerApp] jupyterlab | extension was successfully loaded.
[W 2024-05-15 08:20:47.575 ServerApp] [Jupytext Server Extension] Async contents managers like AsyncLargeFileManager are not supported at the moment (https://github.com/mwouts/jupytext/issues/1020). We will derive a contents manager from LargeFileManager instead.
[I 2024-05-15 08:20:47.575 ServerApp] [Jupytext Server Extension] Deriving a JupytextContentsManager from LargeFileManager
[I 2024-05-15 08:20:47.576 ServerApp] jupyterlab_jupytext | extension was successfully loaded.
[I 2024-05-15 08:20:47.577 ServerApp] notebook | extension was successfully loaded.
[I 2024-05-15 08:20:47.577 ServerApp] Serving notebooks from local directory: /media/creativeucon/D/repos/uconcreative/accent-coach/llmware
[I 2024-05-15 08:20:47.577 ServerApp] Jupyter Server 2.14.0 is running at:
[I 2024-05-15 08:20:47.577 ServerApp] http://localhost:8888/tree?token=25f2f372a27880eab38b197a92be0c8e34cdd878678892d4
[I 2024-05-15 08:20:47.577 ServerApp]     http://127.0.0.1:8888/tree?token=25f2f372a27880eab38b197a92be0c8e34cdd878678892d4
[I 2024-05-15 08:20:47.577 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2024-05-15 08:20:47.590 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///home/creativeucon/.local/share/jupyter/runtime/jpserver-5648-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/tree?token=25f2f372a27880eab38b197a92be0c8e34cdd878678892d4
        http://127.0.0.1:8888/tree?token=25f2f372a27880eab38b197a92be0c8e34cdd878678892d4
[I 2024-05-15 08:20:47.600 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[W 2024-05-15 08:21:01.034 ServerApp] 404 GET /api/kernels/60e5df90-3bc7-480e-9648-0b8d933fd57f?1715728861032 (127.0.0.1): Kernel does not exist: 60e5df90-3bc7-480e-9648-0b8d933fd57f
[W 2024-05-15 08:21:01.034 ServerApp] wrote error: 'Kernel does not exist: 60e5df90-3bc7-480e-9648-0b8d933fd57f'
    Traceback (most recent call last):
      File "/opt/jupyter-venv/lib/python3.12/site-packages/tornado/web.py", line 1790, in _execute
        result = await result
                 ^^^^^^^^^^^^
      File "/opt/jupyter-venv/lib/python3.12/site-packages/jupyter_server/auth/decorator.py", line 73, in inner
        return await out
               ^^^^^^^^^
      File "/opt/jupyter-venv/lib/python3.12/site-packages/jupyter_server/services/kernels/handlers.py", line 75, in get
        model = await ensure_async(km.kernel_model(kernel_id))
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/opt/jupyter-venv/lib/python3.12/site-packages/jupyter_server/services/kernels/kernelmanager.py", line 501, in kernel_model
        self._check_kernel_id(kernel_id)
      File "/opt/jupyter-venv/lib/python3.12/site-packages/jupyter_server/services/kernels/kernelmanager.py", line 532, in _check_kernel_id
        raise web.HTTPError(404, "Kernel does not exist: %s" % kernel_id)
    tornado.web.HTTPError: HTTP 404: Not Found (Kernel does not exist: 60e5df90-3bc7-480e-9648-0b8d933fd57f)
[W 2024-05-15 08:21:01.046 ServerApp] 404 GET /api/kernels/60e5df90-3bc7-480e-9648-0b8d933fd57f?1715728861032 (51fba85bb9da4f7a9148881fd8a8289b@127.0.0.1) 11.81ms referer=http://localhost:8888/notebooks/examples/Datasets/build_aws_transcript_dataset.ipynb
[W 2024-05-15 08:21:26.187 ServerApp] Notebook examples/Datasets/working_with_datasets.ipynb is not trusted
[W 2024-05-15 08:21:26.610 ServerApp] Notebook examples/Datasets/working_with_datasets.ipynb is not trusted
[I 2024-05-15 08:21:28.973 ServerApp] Kernel started: 2f72c1d9-104a-4916-a183-b6f23533b2fb
[I 2024-05-15 08:21:29.324 ServerApp] Connecting to kernel 2f72c1d9-104a-4916-a183-b6f23533b2fb.
[I 2024-05-15 08:21:29.337 ServerApp] Connecting to kernel 2f72c1d9-104a-4916-a183-b6f23533b2fb.
[I 2024-05-15 08:21:29.350 ServerApp] Connecting to kernel 2f72c1d9-104a-4916-a183-b6f23533b2fb.
[I 2024-05-15 08:21:30.941 ServerApp] Connecting to kernel 2f72c1d9-104a-4916-a183-b6f23533b2fb.
[I 2024-05-15 08:23:26.658 ServerApp] Saving file at /examples/Datasets/working_with_datasets.ipynb
[I 2024-05-15 08:23:26.658 ServerApp] Saving working_with_datasets.ipynb
[I 2024-05-15 08:23:40.308 ServerApp] Starting buffering for 2f72c1d9-104a-4916-a183-b6f23533b2fb:c0ef96dc-fa99-40cd-a395-67c26b0c2ba3
